server/src/test/java/org/elasticsearch/cluster/metadata/DataStreamTests.java

```diff
@@ -54,7 +54,7 @@ protected DataStream createTestInstance() {
     public void testRollover() {
         DataStream ds = DataStreamTestHelper.randomInstance().promoteDataStream();
         Index newWriteIndex = new Index(getDefaultBackingIndexName(ds.getName(), ds.getGeneration() + 1), UUIDs.randomBase64UUID(random()));
-        DataStream rolledDs = ds.rollover(newWriteIndex);
+        DataStream rolledDs = ds.rollover(newWriteIndex, DataStream.NEW_FEATURES_VERSION);
```

Comment:
maybe also add a test here with a version before 7.11?

Reply:
and maybe also a test to `MetadataCreateDataStreamServiceTests` that tests creating a data stream when minimum node version is before 7.11?